py3status/modules/external_script.py

```diff
@@ -75,10 +75,12 @@ def external_script(self):
             output = e.output or e.error
             self.py3.error(output)
 
-        output_text = output_lines[0]
-
-        if self.strip_output:
-            output_text = output_text.strip()
+        if output_lines:
```

Comment:
This variable can be undefined if `CommandError` is raised.

Reply:
Nice catch. High :five:. I optimized-pushed.

Reply:
I prefer the old PR

Reply:
K. Old PR w/ column fix.